add support examples for aarch64 (#35)
suse-edge/atip: telco-examples/edge-clusters/airgap/README.md

There are two steps to deploy an edge cluster:

> - Create the image for the edge cluster using the combination of a Kiwi process (to create a base image) + Edge Image Builder (to customize it) in order to prepare all the package dependencies and the requirements for the edge cluster.

Reviewer: ```suggestion
- Create the image for the edge cluster with Kiwi (to create the base image) + Edge Image Builder to customize it including all the packages, dependencies and requirements.
```

You need to modify the following folder:

> - `base-images` - To include inside the raw image generated with the following Kiwi process:

Reviewer: ```suggestion
- `base-images` - To include the raw image generated by Kiwi as:
```

```
mkdir output
sudo podman run --privileged -v /etc/zypp/repos.d:/micro-sdk/repos/ -v $(pwd)/output:/tmp/output -it registry.opensuse.org/isv/suse/edge/factory/images_16.0/kiwi-builder:10.2.12.0 build-image -p Base-RT-SelfInstall
```

> In the output folder, you will find the raw image to be copied to the base-image folder and used as a reference in the `eib/telco-edge-airgap-cluster.yaml` file.

Reviewer: ```suggestion
The resulting raw image needs to be copied over to the `base-image` folder and used as a reference in the `eib/telco-edge-cluster.yaml` file:

```
cp $(pwd)/output/*.raw base-images/
```
```

I'm not sure why GitHub doesn't like the format

> > **_Note:_** For more information about this process you can follow the [full guide instructions in official docs](https://documentation.suse.com/suse-edge/3.2/html/edge/guides-kiwi-builder-images.html)

Reviewer: ```suggestion
> **_Note:_** For more information, read the "[Building Updated SUSE Linux Micro Images with Kiwi](https://documentation.suse.com/suse-edge/3.2/html/edge/guides-kiwi-builder-images.html)" official documentation.
```

Author: same than previous ones
